src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -1026,17 +1105,34 @@ private void doDeleteSwitchRules(final CommandMessage message) {
                 Integer server42Vlan = request.getServer42Vlan();
                 MacAddress server42MacAddress = request.getServer42MacAddress();
                 if (request.isServer42FlowRttFeatureToggle()) {
-                    switchManager.installServer42TurningFlow(dpid);
+                    switchManager.installServer42FlowRttTurningFlow(dpid);
 
-                    if (request.isServer42FlowRttSwitchProperty() && server42Port != null && server42Vlan != null
-                            && server42MacAddress != null) {
-                        switchManager.installServer42OutputVlanFlow(
+                    if (request.isServer42FlowRttSwitchProperty()
+                            && server42Port != null && server42Vlan != null && server42MacAddress != null) {
+                        switchManager.installServer42FlowRttOutputVlanFlow(
                                 dpid, server42Port, server42Vlan, server42MacAddress);
-                        switchManager.installServer42OutputVxlanFlow(
+                        switchManager.installServer42FlowRttOutputVxlanFlow(
                                 dpid, server42Port, server42Vlan, server42MacAddress);
 
-                        for (Integer port : request.getServer42FlowRttPorts()) {
-                            switchManager.installServer42InputFlow(dpid, server42Port, port, server42MacAddress);
+                        if (request.isMultiTable()) {
```

Comment:
I remembered why we do not need here `isMultitable()`. Case:
1. switch is in multi table mode.
2. you create a flow with server42 on it.
3. Kilda installs flow rtt input rule
4. You update switch property to set multitable = `False`
5. But flow endpoint still in multi table mode. We have such flags in FlowPath. Flow will become single table only after reroute. 

So you need flow rtt input rule for such flow even if switch is in single table mode. 

Please see `SwitchRuleServiceImpl.fillFlowPorts()`

Reply:
Ok. I've reverted the changed related to Flow RTT and multitable.